src/frontend/src/lib/derived/network.derived.ts

```diff
@@ -19,9 +20,19 @@ export const networkId: Readable<NetworkId> = derived(
 			: DEFAULT_NETWORK_ID
 );
 
+export const selectedNetwork: Readable<Network> = derived(
```

Comment:
no, it didn't change, just moved: I wanted to use it in networkTokens and it didn't allow me because the declaration was done after (it raised an error)

Reply:
Ah ok!